code/game/machinery/doors/airlock.dm

```diff
@@ -138,6 +127,16 @@
 	/// Used for papers and photos pinned to the airlock
 	var/note_overlay_file = 'icons/obj/doors/airlocks/station/overlays.dmi'
 
+	var/mask_file = 'icons/obj/doors/airlocks/mask_32x32_airlocks.dmi' // because filters aren't allowed to have icon_states :(
+	var/mask_x = 0
+	var/mask_y = 0
+	var/anim_parts = "left=-14,0;right=13,0"
+	var/list/part_overlays
+	var/panel_attachment = "right"
+	var/note_attachment = "left"
+	var/mask_filter
+	var/static/list/airlock_overlays = list()
```

Comment:
Autodoc

Reply:
removed, it's back in the get_airlock_overlay() proc